src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -103,6 +105,14 @@ protected boolean startProcessDocs(final RootDoc rootDoc) throws IOException {
             if (options[0].equals(OUTPUT_FILE_EXTENSION_OPTION)) {
                 outputFileExtension = options[1];
             }
+            if (options[0].equals(INDEX_FILE_EXTENSION_OPTION)) {
+                indexFileExtension = options[1];
+            }
+        }
+
+        // default to outputFileExtension
+        if (indexFileExtension == null) {
+            indexFileExtension = outputFileExtension;
```

Comment:
I did it like that to assing the value of `-output-file-extension` to the indexFileExtension in case that the `-index-file-extension` option is not provided. I think that this is better than provide the two of them in the case that they should be the same.

I kept this and assign `indexFileExtension` to null before parsing options to maintain this behavior. If you want me to change it, just let me know...